packages/network/networkNames.md

```diff
@@ -0,0 +1,133 @@
+# Network Names
+
+Though networks have proper full names, we sometimes need to refer to a chain by a name slug.
+
+A name slug is the name, sometimes simplified, and in kebab-case. This maximizes compatibility in the event that you want to use chain names as url paths, or filenames.
+
+You can use this file as a reference to look up the correct network name slug a network's full name or its chain ID.
+
+| Chain ID | Name Slug |
+| --- | --- |
+| 1 | mainnet |
+| 3 | ropsten |
+| 4 | rinkeby |
+| 5 | goerli |
```

Comment:
Some of these are obsolete / unsupported chains.

I think we should mark them as such.

Reply:
Yeah, now that this doc is autogenerated from the pacakge/network/constants, it should either be removed there or marked somehow as deprecated so we can omit it from this new doc. Or we could leave it for now, not a rush

Reply:
@bunnybones1 They are already marked as deprecated, see here: https://github.com/0xsequence/devops/blob/b0f519669e4ec25963125c2ba0618d8d1c5a360f/constants.ts#L222

Can you have the script parse this flag and ignore if it's `deprecated: true`

Reply:
done!